cluster/cluster.go

```diff
@@ -67,6 +68,10 @@ type Cluster interface {
 
 	RemoveToDeleteBatch(batch *ToDeleteBatch) error
 
+	CheckConstantReplicationFactor(expectedReplicationFactor int) error
```

Comment:
We don't need the functions on the interface here. It's ok they're simply exported from Dragon

Reply:
Removed them